### ISSUE TITLE
problem downloading Sentinel-3 data

### ISSUE THREAD

**Issue by sshbb** [sentinelsat/sentinelsat#410]
I use the script below to download Sentibel-3 data. For some reason, the API does not work for requests after 2020.  Any recommendations?

```python
from sentinelsat import SentinelAPI, read_geojson, geojson_to_wkt
from datetime import date
api = SentinelAPI(‘MYUSER’, ‘MYPASS’, ‘httpsscihub.copernicus.eudhus#home’)
products = api.query(filename = ‘S3A_SR_2_LAN____LN3_O_NT_003’, date=(‘2020-01-01T000000.000Z’, ‘2020-11-01T000000.000Z’))
api.download_all(products, ‘DIRECTORY’)
```

**Comment by j08lue** [comment]
Not sure whether this is the cause of the issue, but for finding Sentinel-3 data, rather than `filename`, you should use `platformname="Sentinel-3"`, perhaps combined with `instrumentshortname` and/or `producttype`.

**Comment by diogovad** [comment]
Following on this. I successfully downloaded Sentinel-2 data and Sentinel 3 (e.g. OL_2_LFR___). However, when I try to download Sentinel-3 OL_1_EFR___ it just throws StopIteration error. I have used the same footprint and have tried different temporal intervals with no sucess.

**Comment by j08lue** [comment]
@diogovad Please provide a minimal *working* example (just omit the credentials but please do include an area of interest) that reproduces this issue.

**Comment by diogovad** [comment]
```
import numpy as np 

import time
import datetime
from sentinelsat import SentinelAPI, read_geojson,geojson_to_wkt

geojsonfilename='/path/to/geojson/file.geojson'

yyyystart, mmstart, ddstart = 2020, 10, 25
yyyyend, mmend, ddend = 2020, 11, 25

start_date = datetime.datetime(yyyystart, mmstart, ddstart)
end_date= datetime.datetime(yyyyend, mmend, ddend)

api = SentinelAPI(user="USER", password="PASS")

footprint = geojson_to_wkt(read_geojson(geojsonfilename))


products = api.query(footprint,
                     date = (start_date, end_date), #date=('20151219', date(2015, 12, 29))
                     producttype= 'OL_2_LFR___',
                     cloudcoverpercentage=(0, 80))


api.download_all(products)
```

the footprint wkt: MULTIPOLYGON(((-9.8895 39.2625,-8.3741 39.2590,-8.3924 38.1262,-9.8895 38.1298,-9.8895 39.2625)))

In my case, if I use producttype OL_1_EFR___ it does not work, while it works with OL_2_LFR.

Thank you,

**Comment by valgur** [comment]
Should be fixed with #378. Feel free to re-open if that's not the case.
